openpower/configs/swift_defconfig

```diff
@@ -0,0 +1,76 @@
+BR2_OPENPOWER_CONFIG_NAME="swift"
+BR2_HOSTBOOT_CONFIG_FILE="swift.config"
+BR2_OPENPOWER_MACHINE_XML_VERSION="14a1bec7d32fabf990338cc6c7a33df99d86f499"
+BR2_OPENPOWER_TARGETING_ECC_FILENAME="SWIFT_HB.targeting.bin.ecc"
+BR2_OPENPOWER_TARGETING_BIN_FILENAME="SWIFT_HB.targeting.bin"
+BR2_HOSTBOOT_BINARY_WINK_FILENAME="p9a.ref_image.hdr.bin.ecc"
+BR2_HOSTBOOT_BINARY_SBE_FILENAME="axone_sbe.img.ecc"
+BR2_OPENPOWER_PNOR_FILENAME="swift.pnor"
+BR2_OPENPOWER_PNOR_XML_LAYOUT_FILENAME="axonePnorLayout_64.xml"
+BR2_OPENPOWER_BIOS_XML_FILENAME="SWIFT_bios.xml"
+BR2_OPENPOWER_MRW_XML_FILENAME="SWIFT_hb.mrw.xml"
+BR2_OPENPOWER_SYSTEM_XML_FILENAME="SWIFT_hb.system.xml"
+BR2_OPENPOWER_MACHINE_XML_FILENAME="swift.xml"
+BR2_OPENPOWER_MACHINE_XML_CUSTOM_GIT_VALUE="git@github.com:open-power/swift-xml.git"
+BR2_OPENPOWER_MACHINE_XML_CUSTOM_GIT=y
+BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="swift-xml"
```

Comment:
You shouldn't need the custom settings by default. I think it was used in the wrapper to point somewhere other than the default. 

Witherspoon used:
BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="witherspoon-xml"
BR2_OPENPOWER_MACHINE_XML_VERSION="0f9b3666becbd5a94d31ff39a2f2a81260c961ed"
BR2_OPENPOWER_MACHINE_XML_FILENAME="witherspoon.xml"
BR2_OPENPOWER_SYSTEM_XML_FILENAME="WITHERSPOON_hb.system.xml"
BR2_OPENPOWER_MRW_XML_FILENAME="WITHERSPOON_hb.mrw.xml"
BR2_OPENPOWER_BIOS_XML_FILENAME="WITHERSPOON_bios.xml"

And no use of custom.

Reply:
Thanks.  I will remove the XML_CUSTOM lines, but keeping the other CUSTOM lines that witherspoon_defconfig also uses.